src/main/java/uk/gov/companieshouse/model/validator/AppealReasonValidator.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.companieshouse.model.validator;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.exception.AppealReasonException;
+import uk.gov.companieshouse.model.Reason;
+
+@Component
+public class AppealReasonValidator {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(AppealReasonValidator.class);
+
+    public void validate(Reason reason) throws AppealReasonException {
+        if(reason.getOther() == null ^ reason.getIllnessReason() == null){
```

Comment:
Isn't this logic basically identical to 
`(reason.getReasonType() != null)`
Would there ever be a reason that we don't wont the validation criteria to be the same?

Reply:
Awesome! done